althea_kernel_interface/Cargo.toml

```diff
@@ -14,6 +14,7 @@ log = "0.4"
 serde_derive = "1.0"
 serde = "1.0"
 althea_types = { path = "../althea_types" }
+ipnetwork = "0.18"
```

Comment:
this is the latest version of ipnetwork, please update it across all the Cargo.toml in the repo (nearly all of our crates depend on it)

Reply:
It already seems to be 0.18 everywhere?